.github/workflows/CI.yml

```diff
@@ -149,14 +149,13 @@ jobs:
       run: ./interop/test.sh --use_tls tls_rustls
       shell: bash
 
-  semver:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - uses: obi1kenobi/cargo-semver-checks-action@v2
-      with:
-        feature-group: all-features
-        exclude: grpc
+  # semver:
```

Comment:
Will re-enable once we get the release out.

Reply:
Can `if: false` condition be used to disable this job so as to maintain it instead of commenting out.